bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx66.html

```diff
@@ -0,0 +1,182 @@
+{# License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+  {% from "macros-protocol.html" import hero, call_out_compact with context %}
+
+  {% from "macros.html" import fxa_email_form, fxa_link_fragment, google_play_button with context %}
+
+  {% add_lang_files "firefox/whatsnew_66" %}
+
+  {% extends "firefox/base-protocol.html" %}
+
+  {% block page_title %}{{_('Get a Firefox Account – Keep your data private, safe and synced') }}{% endblock %}
+  {% block page_desc %}{{_('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.') }}{% endblock %}
+
+  {% block page_css %}
+   {{ css_bundle('firefox_whatsnew_66') }}
+  {% endblock %}
+
+  {% set fxa_utm_params = {
+    'campaign': 'wnp66',
+    'content': '/firefox/WNP/66/',
+    'source': 'wnp-66'
+  } %}
+
+  {% block body_class %}state-fxa-default{% endblock %}
+
+  {% block site_header %}{% endblock %}
+
+  {% block content %}
+
+  <main class="mzp-t-firefox">
+    <header class="c-page-header">
+      <div class="mzp-l-content c-page-header-inner">
+        {{ high_res_img('logos/firefox/logo-quantum-wordmark-large.png', {'alt': 'Firefox', 'width': '147', 'height': '55', 'class': 'c-page-header-logo-fx'}) }}
+        <h1 class="c-page-header-up-to-date">{{ _('Congrats! You’re using the latest version of Firefox.') }}</h1>
+        <img src="{{ static('img/logos/mozilla/wordmark-dark.svg') }}" alt="Mozilla" width="112" height="32" class="c-page-header-logo-moz">
+      </div>
+    </header>
+
+    <section class="c-sticky-signup fxa-form-cta">
+        <div class="mzp-l-content">
+          <div class="c-sticky-signup-container">
+            <div class="c-sticky-signup-content">
+              <h2 class="c-sticky-signup-title">{{ _('Get a Firefox Account and all the benefits it unlocks. ') }}</h2>
+              <button type="button" class="sticky-dismiss">Dismiss this prompt.</button>
```

Comment:
This is a hard-coded English string. At this stage we probably need to find an alternative that's in main.lang, such as "Close" https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/main.lang#L168 ?